src/sentry/incidents/models/alert_rule.py

```diff
@@ -47,11 +47,11 @@ def __call__(self, subscription: QuerySubscription, *args: Any, **kwargs: Any) -
         ...
 
 
-alert_subscription_callback_registry: dict[AlertRuleMonitorType, SubscriptionCallback] = {}
+alert_subscription_callback_registry: dict[AlertRuleMonitorTypeInt, SubscriptionCallback] = {}
```

Comment:
Oh hmm - the key will be different so how might this registry be affected?

don't think it should matter, but just want to double check

Reply:
👍 yeah - confirm this is just a name change, the type is still the same (it'll actually be a slightly better type now that it's an IntEnum 😄)